Controllers/FrontController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Controllers;
+
+class FrontController
+{
+    protected $controller;
+    protected $action, $params;
+    static $instance;

```

Comment:
what about visibility of this property? by default php makes it public, but please, declare it.